esp/platform/txsummary.hpp

```diff
@@ -78,9 +91,12 @@ class CTxSummary : extends CInterface
     };
 
     using Entries = std::list<Entry>;
+    using Timers = std::map<StringBuffer, CumulativeTimer*>;
 
-    Entries   m_entries;
-    unsigned  m_creationTime;
+    mutable Monitor m_sync;
```

Comment:
My preference for mutex, even within a single process, is based on recent experience as a Windows developer. We substituted mutex for critical section in all of our projects after observing unexplained deadlocks with critical sections. If critical sections are working reliably, I'll make the change.